File: PRAK601-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
baris, kolom = map(int, input("jumlah baris dan kolom matriks: ").split())


matriks = []

print("Masukkan elemen matriks:")
elemen_input = list(map(int, input().split()))

for i in range(0, len(elemen_input), kolom):
    matriks.append(elemen_input[i:i + kolom])

print("Hasil Matriks:")
for i in range(baris):
    for j in range(kolom):
        print(matriks[i][j], end=" ")
    print()

File: PRAK601-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>

int main() {
    int baris, kolom;

    printf("Jumlah baris dan kolom matriks: ");
    scanf("%d %d", &baris, &kolom);

    int matriks[baris][kolom];

    printf("Elemen matriks:\n");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }

    printf("Hasil Matriks:\n");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }

    return 0;
}
